src/plugins/persistence/couch/pluginSpec.js

```diff
@@ -28,7 +28,7 @@ import {
 describe('the plugin', () => {
     let openmct;
     let provider;
-    let testPath = '/test/db';
+    let testPath = 'http://localhost:9990/openmct';
```

Comment:
my approach with tests will be to:

- ensure worker comes up
- test couch sse connection using stubbed events